types/ember-cli-test-loader/test-support/index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'ember-cli-test-loader/test-support/index' {
```

Comment:
Should we port these to the library in question? Yaks on yaks on yaks.

Reply:
Can either merge this as is or release https://github.com/ember-cli/ember-cli-test-loader/pull/77 and switch to using that.